Datastore/tests/Unit/TransactionTest.php

```diff
@@ -104,47 +132,26 @@ public function testLookup(callable $transaction)
         $this->assertEquals($this->key->keyObject(), $res->key()->keyObject());
     }
 
-    public function testLookupWithReadTime()
```

Comment:
Removing this test as it's not a valid scenario.

> Transactions in Datastore are used for ensuring consistency and isolation, 
but they do not provide the ability to specify a specific read time.
Transactions guarantee that all reads and writes within the transaction are
consistent with each other and appear to have occurred at a single point
in time, but the exact timestamp of that point is not configurable.